src/deepsparse/pipeline.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         self.schedulers = schedulers
         self.pipeline_state = pipeline_state
         self._continuous_batching_scheduler = continuous_batching_scheduler
+        self.timer_manager = TimerManager()
```

Comment:
Im seeing two time_managers here? Line 93 and line 95.

Reply:
Yes good catch